README.md

```diff
@@ -51,13 +51,52 @@ If you don't have a leetcode account you can create one from [here](https://leet
 
 ## 🛠️ Installation Steps
 
-1. Fork the project
-2. Clone the project
-3. Create a new branch (`git checkout -b new-feature`)
-4. Make the appropriate changes in the files (` git add . `)
-5. Add changes to reflect the changes made (`git commit -m 'commit message'`)
-6. Push to the branch (` git push `)
-7. Create a Pull Request
+#### `STEP 1` :  Fork [this](https://github.com/cdhiraj40/LeetDroid.git) repository.
+
+#### `STEP 2` :  Clone your forked copy of the project.
+```
+git clone https://github.com/<your_username>/LeetDroid.git
+```
+#### `STEP 3` : Navigate to the project directory .
+```
+cd resume-builder
+```
+#### `STEP 4` : Add a reference (remote) to the original repository.
+```
+git remote add upstream https://github.com/cdhiraj40/LeetDroid.git
+```
+#### `STEP 5` : Check the remotes for this repository.
+```
+git remote -v
+```
+#### `STEP 6` : Always take a pull from the upstream repository to your master branch to keep it at par with the main project (updated repository).
```

Comment:
This step is not needed because forking and cloning the repository will have the latest changes already.